code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -293,7 +293,7 @@ There are several things that need to be remembered:
 		var/handled_by_bodytype = TRUE
 		var/icon_file
 
-		if(dna.species.bodytype & BODYTYPE_TESHARI)
+		if(dna.species.bodytype & BODYTYPE_CUSTOM)
```

Comment:
why

Reply:
was taking reference from a SR PR that handles both Vox and Teshari, so it looked like it was a simpler option

Reply:
This is objectively not how my limb system works. This will break shit.

Reply:
gotcha, ill revert it back to what it was